test_scripts/Polices/External_UCS/021_ATF_N_TC_PTU_with_disallowed_by_external_consent_entities_off_struct_with_invalid_type_of_params.lua

```diff
@@ -0,0 +1,213 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [Policies] External UCS: PTU with "disallowed_by_external_consent_entities_off" struct with invalid type of params
+--
+-- Description:
+-- In case:
+-- SDL receives PolicyTableUpdate with “disallowed_by_external_consent_entities_off:
+-- [entityType: <any_type_except_Integer>, entityId: <any_type_except_Integer>]”
+-- -> of "<functional grouping>" -> from "functional_groupings" section
+-- SDL must:
+-- a. consider this PTU as invalid
+-- b. do not merge this invalid PTU to LocalPT.
+--
+-- Steps:
+-- 1. Register app
+-- 2. Activate app
+-- 3. Perform PTU (make sure 'disallowed_by_external_consent_entities_off' section is defined
+-- with invalid parameters)
+-- 4. Verify status of update
```

Comment:
@istoimenova, I also agree with you, but we have a lot of scripts on Policies functionality. And such verification (unsuccessful update) is part of these scripts.
All these checks have to be included in scope of regression testing.